src/clients.rs

```diff
@@ -86,22 +115,25 @@ pub async fn create_http_clients(
                 let cert_pem = tokio::fs::read(cert_path).await.unwrap_or_else(|error| {
                     panic!("error reading cert from {cert_path:?}: {error}")
                 });
-                let identity = reqwest::Identity::from_pem(&cert_pem)?;
+                let identity = reqwest::Identity::from_pem(&cert_pem)
+                    .unwrap_or_else(|error| panic!("error creating identity: {error}"));
```

Comment:
Right, the only error that would return here is like an invalid cert, e.g. the format isn't pem or corrupted. This would happen when the service starts up. I'll update the panic message to be more specific.

Reply:
Updated to `error parsing cert: {error}`